File: 1113 - Crescente e Decrescente/crescente_e_decrescente.py
import sys

for linha in sys.stdin:
    partes = linha.split()
    if len(partes) < 2:
        continue
    x, y = int(partes[0]), int(partes[1])
    if x == y:
        break
    if x < y:
        print("Crescente")
    else:
        print("Decrescente")

File: 1113 - Crescente e Decrescente/crescente_e_decrescente.c
#include <stdio.h>

int main(){
    int x, y;
    while(scanf("%d %d", &x, &y) == 2){
        if(x == y) break;
        if(x < y) printf("Crescente\n");
        else printf("Decrescente\n");
    }
    return 0;
}
